### DISCUSSION TITLE
The type 'Color' was not found

### DISCUSSION THREAD

**Discussion by WeylinRG** [unoplatform/uno#17603] [Q&A]
Hi,

I'm new to Uno and I'm trying to run a project that I have just created. But I have immediately come across a problem. I have error messages, that the type 'Color' was not found in the file ColorPaletteOverride.xaml. The app fails to run because of it. How can I fix it?

![изображение](https://github.com/user-attachments/assets/41c77d67-930b-470b-9d2f-6ffd6214612d)

**Comment by jeromelaban** [comment]
Thanks for the report. This is an issue that we are tracking with Microsoft at this time in Visual Studio, where you need to select the `net8.0-windows10.xxx` target in the top left selector of your window (the one that mentions `net8.0-android`).

**Reply by WeylinRG** [reply]
The errors "The type 'Color' was not found" really disappeared, when I followed your advise, but the main problem is that I still can't run the app in the android emulator. It throws an exception "System.Reflection.TargetInvocationException" (you can see in my gif how I do it). But if I remove the section <utum:MaterialToolkitTheme ColorOverrideSource="ms-appx:///Styles/ColorPaletteOverride.xaml"> from App.xaml (that is related to the file ColorPaletteOverride.xaml with this missing 'Color' type), the app starts successfully in the android emulator. Is there a way to fix it and run the app properly in the android emulator?

![uno_android_error](https://github.com/user-attachments/assets/79952f8d-3c0c-4b26-82e7-e64cabfbcb3c)

**Reply by jeromelaban** [reply]
This is a different issue. Are you able to run an app with a default template?

**Reply by kazo0** [reply]
@WeylinRG are you able to upload this sample app as a .zip and I can take a look at this crash on my side?

**Comment by ArchieCoder** [comment]
@jeromelaban I have the warnings despite selecting net8.0-windows10.0.19041

![image](https://github.com/user-attachments/assets/2a21093c-e55d-467a-84e2-06b26f0d773e)
